### PR TITLE
addpkg: trojan-go-git

### PR DESCRIPTION
**Pull request by DuckSoft**
https://github.com/p4gefau1t/trojan-go

### CODE REVIEW

File: archlinuxcn/trojan-go-git/lilac.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+from lilaclib import *
+
+def post_build():
+    git_add_files('PKGBUILD')
+    git_commit()
+    update_aur_repo()
```
Comment (imlonghao) [inline archlinuxcn/trojan-go-git/lilac.py:8]: Don't update AUR repo since it's a VCS package.

Comment (DuckSoft) [reply archlinuxcn/trojan-go-git/lilac.py:8]: I maintain that package on AUR and have granted `lilac` as comaintainer.

Comment (oldherl) [reply archlinuxcn/trojan-go-git/lilac.py:8]: Per AUR Guidelines, you should not update PKGBUILD if it's only a VCS change.

Comment (DuckSoft) [reply archlinuxcn/trojan-go-git/lilac.py]: Done. `lilac.py` removed. 
I picked up this bad habit from `qv2ray-dev-git` package. Later I will file another PR to fix that.

**Comment by DuckSoft** [comment]
NOTE: Please squash this PR. Thank you.

File: archlinuxcn/trojan-go-git/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
+pkgname=trojan-go-git
+_pkgname=trojan-go
+pkgver=0.6.0.r0.gb93b95f
+pkgrel=1
+pkgdesc="A Trojan proxy written in Go (git version)."
+arch=(x86_64)
+url="https://github.com/p4gefau1t/trojan-go"
+license=(GPL3)
+depends=(glibc v2ray-geoip v2ray-domain-list-community)
+makedepends=(go)
```
Comment (imlonghao) [inline archlinuxcn/trojan-go-git/PKGBUILD:10]: Build failed.

```
==> Validating source files with sha512sums...
    trojan-go ... Skipped
==> ERROR: Cannot find the git package needed to handle git sources.
```

**Comment by yan12125** [comment]
FWIW, for VCS packages, update_aur_repo() will not push changes unless there are changes other than pkgver and pkgrel. See https://github.com/archlinuxcn/lilac/pull/147.

File: archlinuxcn/trojan-go-git/PKGBUILD (hunk 2 above)
Comment (DuckSoft) [reply archlinuxcn/trojan-go-git/PKGBUILD]: this is fixed in b12f859e398e9a8a2794b5246138be6f1b50489e.

**Comment by DuckSoft** [comment]
I wonder if this package is ready to go.

**Pull request merged**